service-registry/src/main/java/io/servicecomb/serviceregistry/api/response/FindInstancesResponse.java

```diff
@@ -24,6 +24,10 @@
  * Created by   on 2016/12/8.
  */
 public class FindInstancesResponse {
+  private int statusCode;
+
+  private String rev;
```

Comment:
If rev mean revision, please use revision to explain the variable itself.

Reply:
The field that servicecenter provides is rev,for example: http://127.0.0.1:30100/v4/default/registry/instances?appId=springmvctest&serviceName=springmvc&version=0.0.2&rev=14

Reply:
we can use JsonProperty to map net name to local name

Reply:
is this string type?